### DISCUSSION TITLE
After the update neovim to v0.10, the colors has changed

### DISCUSSION THREAD

**Discussion by optic0n** [neovim/neovim#30301] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/c16167e5-3f8c-4c2d-b36a-1da33dae45e4)
![Screenshot_2024-09-06_12-33-03](https://github.com/user-attachments/assets/cc6ed1b1-a72d-442e-9337-6e204c394572)
After updating Neovim to version 0.10.0, the colors in my setup changed, as shown in the screenshots. On the left screenshot, I am using Arch Linux with NVIM v0.10.0, and on the right, I am using Ubuntu with NVIM v0.7.2. The autocomplete menu color changed, and a white line appeared in the NERDTree plugin. As you can see, the issue doesn't seem to be related to the colorscheme, and running `colorscheme vim` did not resolve it. Any thoughts?

**Comment by zeertzjq** [comment]
What colorscheme are you using?

**Reply by optic0n** [reply]
[iceberg](https://github.com/cocopon/iceberg.vim)

**Reply by zeertzjq** [reply, marked as answer]
Hmm, first you can try `:hi! link WinSeparator VertSplit` for the white line.

I have no idea about the completion menu. Maybe you can consult the coc.nvim repository.

**Reply by optic0n** [reply]
Thank you very much, you fixed the white line issue. Regarding the autocompletion menu, it seems the issue is with `coc.nvim`. Because if I delete this plugin, the old colors return to the completion menu.
